bindings/pyroot_experimental/PyROOT/test/ttree-tchain_setbranchaddress.py

```diff
@@ -0,0 +1,113 @@
+import unittest
+from array import array
+
+import ROOT
+from libcppyy import SetOwnership
+import numpy as np
+
+
+class TTreeTChainSetBranchAddress(unittest.TestCase):
+    """
+    Test for the pythonization of TTree/TChain::SetBranchAddress, which allows to pass proxy
+    references as arguments from the Python side. Example:
+    `v = ROOT.std.vector('int')()`
+    `t.SetBranchAddress("my_vector_branch", v)`
```

Comment:
Are docstrings in Markdown? Or will this appear as \`stuff\` in the documentation?

Reply:
No, they are not. Here I just followed a convention of NumPy style docstrings that encloses variable/module names in backticks to do the same with this piece of code.